### PR TITLE
Update a4a-readme.md

### PR DESCRIPTION
**Pull request by jasti**
Clarifying next steps for A4A interested parties.

### CODE REVIEW

File: ads/google/a4a/docs/a4a-readme.md [hunk 1]
```diff
@@ -28,21 +28,25 @@ AMP ads are designed to work on both AMP and Non-AMP webpages,  including deskto
...
+Here is how you can participate. If you are:
 
 ## Publishers
 
 If publishers want to serve their direct-sold ad formats they must create the ads in[ A4A format](https://github.com/google/amphtml/blob/master/extensions/amp-a4a/amp-a4a-format.md) (or use a creative agency), and deliver them using an AMP ad supported ad server.
 
+The following adservers support serving AMP ads at the moment:
+1. DoubleClick for Publishers
+2. TripleLift 
+
```
Comment (dknecht) [inline ads/google/a4a/docs/a4a-readme.md]: Dianomi

File: ads/google/a4a/docs/a4a-readme.md [hunk 2]
```diff
@@ -28,21 +28,26 @@ AMP ads are designed to work on both AMP and Non-AMP webpages,  including deskto
 
 # Current status
 
-The AMP ads format spec has been [released](https://github.com/google/amphtml/blob/master/extensions/amp-a4a/amp-a4a-format.md) and any creative developer can create AMP ads. In order for ads to get preferred treatment on AMP pages, ad server support is required. Advertisers using DoubleClick or  AdSense can  already deliverprogrammatic AMP ads to publisher AMP pages. Publishers using  DFP (DoubleClick for Publishers) can already deliver their own AMP ads. Advertisers or publishers using other ad providers  can implement AMP ads with the help of a signing service like CloudFlare. Cloudflare provides AMP ad verification services, enabling any independent ad provider to deliver faster, lighter, and more engaging ads.
+The AMP ads format spec has been [released](https://github.com/google/amphtml/blob/master/extensions/amp-a4a/amp-a4a-format.md) and any creative developer can create AMP ads.A number of ad providers are working on automatically converting ads to AMP ads whenever possible. e.g. Adsense
```
Comment (keithwrightbos) [inline ads/google/a4a/docs/a4a-readme.md]: Add spaces between "create AMP ads."  and "A number".  Also I believe the S is capitalized in AdSense

Comment (lannka) [inline ads/google/a4a/docs/a4a-readme.md]: and "." needed after AdSense.

Comment (jasti) [inline ads/google/a4a/docs/a4a-readme.md]: Done.

**Comment by jasti** [comment]
Fixed minor typos. PTAL @lannka

**Comment by lannka** [comment]
I have no idea what the Travis error is about. @erwinmombay do you happen to know?

**Comment by lannka** [comment]
@dvoytenko can you force merge this PR? Presubmit is false alarming.

**Comment by lannka** [comment]
@rsimha-amp FYI, the same false alarm happened again. Do you think it can be a bug?

**Comment by rsimha-amp** [comment]
@lannka This is likely due to an out-of-date branch. See error message in Travis: "If you see a long list of unrelated files below, you may need to sync your branch to master."

Unfortunately, the only fix I'm aware of is to sync the branch to upstream.

**Comment by lannka** [comment]
seems to be a problem with the diff. we should be doing a 3 parties diffing.

**Comment by rsimha-amp** [comment]
@lannka, sent you PR #9166 for review. The clearer error message should alert people when they hit this error.

**Pull request merged**